dvc/dvcfile.py

```diff
@@ -239,6 +259,9 @@ def remove_stage(self, stage):
         else:
             super().remove()
 
+    def merge(self, ancestor, other):
+        raise NotImplementedError
+
```

Comment:
These stubs are for the future, as merging dvcfiles is potentially a handy thing to have.